elements/howto-menu/demo.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+
+<style>
+  howto-menu {
+    position: absolute;
+    display: block;
+    width: 10em;
+  }
+
+  [role="menuitem"] {
+    display: block;
+    width: 100%;
+    padding: .2em .5em;
+    border: 1px solid lightgrey;
+  }
+
+  /* We need this because `display` overrides hidden in this case. */
+  howto-menu[hidden] {
+    display: none;
+  }
+</style>
+
+<div class="text">
+   <button id="menu-btn">options</button>
+   <howto-menu aria-labelledby="menu-btn">
+     <span role="menuitem">first option</span>
+     <span role="menuitem">second option</span>
+     <span role="menuitem">other option</span>
+     <span role="menuitem">yet another option</span>
+   </howto-menu>
+</div>
+
+<script>
+  document.getElementById('menu-btn').addEventListener('click', e => {
+    var menu = document.querySelector('howto-menu');
+    if (menu.hasAttribute('hidden')) {
```

Comment:
I think you can do

```
menu.hidden = !menu.hidden
```

I'm pretty sure `HTMLElement` includes a `hidden` property that also syncs to the attribute automatically.